sdk/network/Microsoft.Azure.Management.Network/src/Microsoft.Azure.Management.Network.csproj

```diff
@@ -10,18 +10,19 @@
 			Development of this library has shifted focus to the Azure Unified SDK. The future development will be focused on "Azure.ResourceManager.Network" (https://www.nuget.org/packages/Azure.ResourceManager.Network/). Please see the package changelog for more information.
 		</Description>
 		<AssemblyName>Microsoft.Azure.Management.Network</AssemblyName>
-		<Version>21.0.0</Version>
+		<Version>21.1.0</Version>
```

Comment:
There are breaking changes, so should increase the major version instead of minor version